app/js/lib/foxbox/settings.js

```diff
@@ -89,8 +87,10 @@ export default class Settings extends Model {
 
   set localOrigin(origin) {
     this._localOrigin = origin ?
-        (new URL(origin)).origin : DEFAULT_LOCAL_ORIGIN;
-    storage.setItem(`${PREFIX}localOrigin`, this._localOrigin);
+        (new URL(origin)).origin : undefined;
```

Comment:
__optional nit:__ can we use ```null``` instead of ```undefined``` - it's initialized with the ```null``` in the constructor (in case we don't have anything in the localStorage it should return ```null```) - just for the consistency.